packages/gafl-webapp-service/src/pages/contact/newsletter/newsletter.njk

```diff
@@ -65,7 +65,7 @@
         {
           value: "no",
           text: mssgs.no,
-          checked: true if not payload or payload['newsletter'] === 'no' else false
+          checked: payload['newsletter'] === 'no'
```

Comment:
Very minor thing but is it worth doing `payload.newsletter` instead to match line 60? Or switching line 60 to match this instead?